modules/sandboxed-containers-rhcos-extensions.adoc

```diff
@@ -0,0 +1,11 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+
+[id="sandboxed-containers-rhcos-extensions_{context}"]
+
+= Red Hat CoreOS extensions
+
+The {sandboxed-containers} uses the {op-system-first} extensions concept. The Kata binaries, RPMs, and QEMU are provided in the form of extensions. They are easily enabled via the `MachineConfig` resources provided by the Machine Config Operator.
+
+For more information on how the extension concept works in practice, see link:https://docs.openshift.com/container-platform/latest/post_installation_configuration/machine-configuration-tasks.html#rhcos-add-extensions_post-install-machine-configuration-tasks[adding extensions to RHCOS].
```

Comment:
You need move this out into the assembly as an additional resource or remove the link. We generally can't have direct links in modules (for now) because we can't have a link from the customer portal to docs.openshift, and we need to keep the user in the same product version.